### PR TITLE
Update ilias harvester

### PR DESCRIPTION
**Pull request by tobiasdierich**
Avoid using SOAP Interface by simulating HTTP requests to ilias.

### CODE REVIEW

File: src/spz/spz/iliasharvester.py [hunk 1]
```diff
@@ -4,60 +4,223 @@
...
+            'ref_id': ref_id,
+        },
+        cookies=cookies,
+        headers=headers
+    )
+    assert r2.status_code == 200
+    text2 = r2.text
+    # these tokens occur multiple times but seem to be unique
+    rtoken = extract_text(text2, "rtoken=", "\"")
+    active_id = extract_text(text2, "active_id=", "&")
```
Comment (Tims777) [inline src/spz/spz/iliasharvester.py]: Different `active_id`s appear in response HTML and we should make sure that the right one is extracted: It seems that the action of 'form#ilToolbar' is exactly what we want so maybe we could extract the `active_id` from there.

Comment (tobiasdierich) [reply src/spz/spz/iliasharvester.py]: Yup, I agree. The token extraction is hacky and unreliable. I copied it from the last ILIAS harvester version :D However, I'll try to figure out something more reliable.

Comment (tobiasdierich) [reply src/spz/spz/iliasharvester.py]: Changed the key extraction part to use an html and url parser.

File: src/spz/spz/iliasharvester.py [hunk 2]
```diff
@@ -16,19 +18,11 @@
 }
 
 
-def extract_text(s, token_begin, token_end):
-    """Extract text from string that is located between 2 tokens."""
-    l0 = len(token_begin)
-    idx0 = s.find(token_begin)
-    assert idx0 > -1
+def get_export_parameters(html):
```
Comment (Tims777) [inline src/spz/spz/iliasharvester.py]: That looks way more robust :)

**Pull request merged**